usermanagment.go

```diff
@@ -163,87 +163,87 @@ type ShareProperties struct {
 
 //ListGroups lists all groups
 func (c *Client) ListGroups() (*Groups, error) {
-	url := umGroups()
+	url := groupsPath()
 	ret := &Groups{}
 	err := c.Get(url, ret, http.StatusOK)
 	return ret, err
 }
 
 //GetGroup gets a group
 func (c *Client) GetGroup(groupid string) (*Group, error) {
-	url := umGroupPath(groupid)
+	url := groupPath(groupid)
 	ret := &Group{}
 	err := c.Get(url, ret, http.StatusOK)
 	return ret, err
 }
 
 //CreateGroup creates a group
 func (c *Client) CreateGroup(grp Group) (*Group, error) {
-	url := umGroups()
+	url := groupsPath()
 	ret := &Group{}
 	err := c.Post(url, grp, ret, http.StatusAccepted)
 	return ret, err
 }
 
 //UpdateGroup updates a group
 func (c *Client) UpdateGroup(groupid string, obj Group) (*Group, error) {
-	url := umGroupPath(groupid)
+	url := groupPath(groupid)
 	ret := &Group{}
 	err := c.Put(url, obj, ret, http.StatusAccepted)
 	return ret, err
 }
 
 //DeleteGroup deletes a group
 func (c *Client) DeleteGroup(groupid string) (*http.Header, error) {
-	url := umGroupPath(groupid)
+	url := groupPath(groupid)
 	ret := &http.Header{}
 	err := c.Delete(url, ret, http.StatusAccepted)
 	return ret, err
 }
 
 //ListShares lists all shares
 func (c *Client) ListShares(grpid string) (*Shares, error) {
-	url := umGroupShares(grpid)
+	url := sharesPath(grpid)
 	ret := &Shares{}
 	err := c.Get(url, ret, http.StatusOK)
 	return ret, err
 }
 
 // GetShare gets a share
 func (c *Client) GetShare(groupid string, resourceid string) (*Share, error) {
-	url := umGroupSharePath(groupid, resourceid)
+	url := sharePath(groupid, resourceid)
 	ret := &Share{}
 	err := c.Get(url, ret, http.StatusOK)
 	return ret, err
 }
 
 // AddShare adds a share
 func (c *Client) AddShare(groupid string, resourceid string, share Share) (*Share, error) {
-	url := umGroupSharePath(groupid, resourceid)
+	url := resourcePath(groupid, resourceid)
```

Comment:
Well, I have no clue about these share-related things but from looking at the previous code I'd expect `sharePath` here. This is also in accordance to the API docs.
Besides that, I am a bit confused that getting and adding a share use the same URL...